lib/util/wrap.js

```diff
@@ -5,7 +5,23 @@
  */
 
 import extend from 'extend'
-import {phrasing} from 'mdast-util-phrasing'
+import {phrasing as hastPhrasing} from 'hast-util-phrasing'
+import {phrasing as mdastPhrasing} from 'mdast-util-phrasing'
+
+/**
+ * @param {MdastNode} node
+ * @returns {boolean}
```

Comment:
You could make this a type guard btw, something along the lines of:

```
* @returns {node is MdastPhrasingContent}
```

should do it I believe? That means the cast later would no longer be needed.

This is a nice to have, not required. If it doesn’t work (well), the changes in this PR look good as is!

Reply:
My JSDoc-fu is inexistent so I just implemented your change.

Reply:
It’s more a TS thing, although written inside JSDoc, the grammar is TS!